main.py

```diff
@@ -69,10 +69,24 @@ def parse_hl(hl):
             lines.append(int(raw_query))
     return lines
 
+def can_track():
```

Comment:
Can you add a docstring ? Perhaps with a link to some doc, not everyone knows about this header :)